src/main/java/ComputerPlayer.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class ComputerPlayer implements Player{
+
+    private Randomizer randomizer;
+    private List<String> locations = new ArrayList();
+    private String mark;
+
+    public ComputerPlayer(Randomizer randomizer, String mark) {
+        this.randomizer = randomizer;
+        for (int cells = 1; cells < 10; cells++) {
+            locations.add(String.valueOf(cells));
+        }
+        this.mark = mark;
+    }
+
+    public String getLocation(InputFeed inputFeed, Board board) {
+        return randomizer.location(availableLocations(board));
+    }
+
+    public List<String> availableLocations(Board board) {
+        for (int option = 0; option < locations.size(); option++) {
+            if(board.get((Integer.parseInt(locations.get(option))) - 1 ) != " ") {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2534705881/issues/source?bid=3284885&fileBranchId=3284887#l24)